Microsoft.Toolkit.Uwp.UI.Controls.Input/TokenizingTextBox/TokenizingTextBox.Properties.cs

```diff
@@ -332,14 +345,12 @@ public string SelectedTokenText
         }
 
         /// <summary>
-        /// Gets or sets how the control should display tokens.
-        /// <see cref="TokenSelectionMode.Multiple"/> is the default. Multiple tokens can be selected at a time.
-        /// <see cref="TokenSelectionMode.Single"/> indicates that only one token can be present in the control at a time.
+        /// Gets or sets the maximum number of token results allowed at a time.
         /// </summary>
-        public TokenSelectionMode TokenSelectionMode
+        public int? MaxTokens
```

Comment:
`int?` is bad for XAML, compile doesn't like it 🙁

See the `MultipleX` property in the `ConstrainedBox` control, you can see how I had to do the same pattern. Use `ReadLocalValue` to check if the developer has set a value or not (in your code above):

https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/ee933854d0013c7856890c1b2b5bec9e3850f908/Microsoft.Toolkit.Uwp.UI.Controls.Primitives/ConstrainedBox/ConstrainedBox.cs#L124

Reply:
I noticed that the default value for the MultipleXProperty DependencyProperty is still `null`. Is that fine or would setting the default value to `DependencyProperty.UnsetValue` be better?

```
 /// <summary>
/// Identifies the <see cref="MaxTokens"/> property.
/// </summary>
public static readonly DependencyProperty MaxTokensProperty = DependencyProperty.Register(
    nameof(MaxTokens),
    typeof(int),
    typeof(TokenizingTextBox),
    new PropertyMetadata(DependencyProperty.UnsetValue, OnMaxTokensChanged));
    // Use DP.UnsetValue intead of null? ^
```

Reply:
Yup, it's very intentionally supposed to be null! 😉